apps/kibana/templates/services/app/dc.yml.j2

```diff
@@ -37,25 +37,48 @@ spec:
           image: "{{ kibana_image_name }}:{{ kibana_image_tag }}"
           imagePullPolicy: Always
           livenessProbe:
-            httpGet:
-              path: /api/status
-              port: {{ kibana_app_port }}
+            exec:
+              command:
+                - /usr/local/bin/status-probe.sh
```

Comment:
About the status reported by `/api/status` : does it reflect the health of kibana only, or also the state of the ES cluster ?

I see here a risk of restarting endlessly the Kibana app if the ES cluster has troubles.
What do you think about keeping the old `livenessProbe` to ensure that the API is responding, and to call the status-probe.sh only in the `readinessProbe` ?

This is only thoughts, I didn't find many information about this status api...

Reply:
Good point! You are right: this is an overall status that includes the ES cluster status (and all plugins). I'll switch to your suggestion.